doc/faq.mkd

```diff
@@ -50,3 +50,16 @@ on your remote repositories:
 [bitbucket-default-branch]: https://answers.atlassian.com/questions/280944/how-to-change-main-branch-in-bitbucket "How to change MAIN branch in BitBucket?"
 [gitolite-v2-default-branch]: http://stackoverflow.com/questions/7091599/git-default-remote-branch-with-gitolite "git default remote branch with gitolite"
 [gitolite-v3-default-branch]: http://stackoverflow.com/questions/13949093/git-change-default-branch-gitolite "git change default branch (gitolite)"
+
+#### I just installed r10k and I'm getting the error "undefined method `keys' for nil:NilClass"
```

Comment:
This is a bug (https://tickets.puppetlabs.com/browse/RK-4) is due to a missing `sources` key in r10k.yaml, and is not related to deploy keys.